backend/service.bal

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) 2025 WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License. 
+import web_app_marketplace.authorization;
+import web_app_marketplace.database;
+import web_app_marketplace.people;
+
+import ballerina/cache;
+import ballerina/http;
+import ballerina/log;
+
+final cache:Cache cache = new ({
+    capacity: 200,
+    defaultMaxAge: 86400.0,
+    cleanupInterval: 900.0
```

Comment:
We don't need both. And let's add `evictionFactor` to determine how the entries will be evicted one the cache is full.
```suggestion
    defaultMaxAge: 86400.0,
    evictionFactor: 0.2
```